### PR TITLE
Remove unsupported split_equally for update_expense

### PR DESCRIPTION
**Pull request by jas14**
`update_expense` doesn't currently support the "split equally" method.

### CODE REVIEW

**Review by jhottenstein — APPROVED** [review]

File: docs/index.html [hunk 1]
```diff
@@ -2224,8 +2224,8 @@
...
+<p>If you are developing a non-commercial plugin application or personal project, we recommend you make use of the Self-Serve API documented here under the API Terms Of Use. Please be aware that our Self-Serve API has conservative rate and access limits, which are subject to change at any time and not well suited to commercial projects. If this is a problem for your use case, please contact us at <a href="mailto:&#x64;&#x65;&#x76;&#x65;&#108;&#111;&#112;&#101;&#x72;&#115;&#64;&#x73;&#112;&#108;&#x69;&#116;&#119;&#x69;&#115;&#x65;&#x2e;&#x63;&#111;&#109;">&#x64;&#x65;&#x76;&#x65;&#108;&#111;&#112;&#101;&#x72;&#115;&#64;&#x73;&#112;&#108;&#x69;&#116;&#119;&#x69;&#115;&#x65;&#x2e;&#x63;&#111;&#109;</a> to discuss your needs.</p>
```
Comment (jhottenstein) [inline docs/index.html]: Why this change?  (and why the html codes?)

Comment (jas14) [reply docs/index.html]: I can only assume a new version of `redoc-cli`. I'll have to properly set up an npm/yarn project.
